packages/core/src/types/asyncapi.ts

```diff
@@ -417,23 +389,22 @@ const Schema: NodeType = {
     default: null,
     readOnly: { type: 'boolean' },
     writeOnly: { type: 'boolean' },
-    // xml: 'Xml',
     examples: { type: 'array' },
     example: { isExample: true },
     deprecated: { type: 'boolean' },
     const: null,
     $comment: { type: 'string' },
-    dependencies: { type: 'object' }, // TODO
+    additionalItems: (value: any) => {
+      return typeof value === 'boolean' ? { type: 'boolean' } : 'Schema';
+    },
+    dependencies: 'PropertyDependency',
```

Comment:
Don't you mind renaming it to just `Dependencies`?

```suggestion
    dependencies: 'Dependencies',
```